bwproject.py

```diff
@@ -164,16 +168,19 @@ def bare_request(self, verb, address_root, address_suffix, access_token="", para
             The response json
         """
         time.sleep(.5)
-        if access_token:
-            params["access_token"] = access_token
 
+        headers = {}
+
+        if access_token:
+            headers["Authorization"]="Bearer {}".format(access_token)
         if data == {}:
-            response = verb(address_root + address_suffix, params=params)
+            response = verb(address_root + address_suffix, params=params, headers=headers)
         else:
+            headers["Content-type"] = "application/json"
             response = verb(address_root + address_suffix,
                             params=params,
                             data=data,
-                            headers={"Content-type": "application/json"})
```

Comment:
You are dropping the content-type header in you changes. I don't know if this matters, but I'd suggest keeping it just in case it does.